srcpkgs/ibus-chewing/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ibus-chewing'
+pkgname=ibus-chewing
+version=1.5.1
+revision=1
+archs="x86_64"
+build_wrksrc=ibus-chewing-1.5.1
+build_style=cmake
+configure_args="-DLIBEXEC_DIR=/usr/libexec"
+make_cmd=make
+hostmakedepends="git cmake pkg-config findutils gettext-devel"
+makedepends="libchewing-devel ibus-devel gob2 libX11-devel gtk+-devel glib-devel"
```

Comment:
I didn't notice this earlier, sorry, but I don't believe new gtk2 packages will be accepted.

Reply:
Ya, that's why I directly cloned the repo before, instead of downloading the realeases.

Reply:
You can get a tarball at a specific commit from GitHub. I don't think this meets quality requirements, it either uses GTK2 or it doesn't have a release.

Reply:
I just simply fork it, and release the latest commit by myself.

Reply:
No, that's equivalent to just not using a release. A release is required because the developers know the best when the code is stable and usable. If we packaged the latest commit, we would have to figure out issues on our own and the maintenance burden would be much higher.

Reply:
Oh, ok then...  What should I do?

Reply:
It's not acceptable right now.

Reply:
you could ask upstream to make a release

Reply:
All right then, sorry for wasting you guys so much time.